Dockerfile

```diff
@@ -1,4 +1,11 @@
-FROM node:12-alpine
+FROM node:15.14.0@sha256:608bba799613b1ebf754034ae008849ba51e88b23271412427b76d60ae0d0627 AS builder
+
+COPY --chown=node:node . /home/node/app
```

Comment:
There is a future optimisation to be done here:
- copy the `package.json` and `lock` files, run `yarn install`
- copy the rest of the files and run `yarn build`

Updating packages happens less frequently than updating code, so it's a good idea to split them out